src/systemd/sd-journal.h

```diff
@@ -156,13 +158,13 @@ int sd_journal_has_persistent_files(sd_journal *j);
         if (sd_journal_seek_tail(j) < 0) { }                            \
         else while (sd_journal_previous(j) > 0)
 
-/* Iterate through the data fields of the current journal entry */
+/* Iterate through all accessible data fields of the current journal entry */
```

Comment:
here you use "accessible" rather than "good". Maybe use "supported" in both cases?